.pipelines/cni/cilium/cilium-cni-load-test.yaml

```diff
@@ -111,16 +111,18 @@ stages:
                 echo "Scale up the pods and immediated restart the nodes"
                 make -C ./hack/swift set-kubeconf AZCLI=az CLUSTER=${RESOURCE_GROUP}
                 make -C ./hack/swift azcfg AZCLI=az REGION=$(LOCATION)
+                cd test/integration/load
                 echo "Scaling the pods down to 100 per node"

```

Comment:
I've missed this previously, but do we want to scale to 100 or 1000?

Reply:
We want to scale it 100 per node. We have 10 nodes so 1000 pods in  total.